validator/client/validator_attest.go

```diff
@@ -113,18 +109,27 @@ func (v *validator) AttestToBlockHead(ctx context.Context, slot uint64) {
 	// TODO(#1366): Use BLS to generate an aggregate signature.
 	attestation.AggregateSignature = []byte("signed")
 
+	log.WithField(
+		"blockRoot", fmt.Sprintf("%#x", attData.BeaconBlockRootHash32),
```

Comment:
Can you add this field to the next log statement below and have one log for this action?